internal/ui/dialog/delete.go

```diff
@@ -45,11 +45,25 @@ func ShowDelete(styles config.Dialog, pages *ui.Pages, msg string, ok okFunc, ca
 	f.AddCheckbox("Force:", force, func(_ string, checked bool) {
 		force = checked
 	})
+
+	confirmCheck := ""
+	if challenge {
+		f.AddInputField("Confirm with \"yes, I want to delete\":", confirmCheck, 30, nil, func(text string) {
```

Comment:
It may make sense to actually pass in the challenge string ie if set add the form field and validate. Then we can reuse this functionality to namespace to fix the original PR. 
Also not sure how much we want to make folks type ie challenge: `Sanitize me!` or a simple `Yes!' might be enough to disrupt the flow and make users stop/think??

Reply:
I made the changes, and at least for deletion of nodes, it is now "delete!"

Do we want to make this default behaviour, or do we want to add a configuration toggle for this?

Reply:
Thank you for the updates Hans! Good point!!
I think doing this on a per resource basis would be a bit much. In k9sAlpha, I have the concept of an `idiotLight`. When this config is set, then confirm and deletion dialogs surface a new field aka `Confirm` with a challenge phrase. I think it would be a good idea to do the same here so folks can opt-in/out for additional guardrails. Would this make sense?

Reply:
Agreed, will look into it when i have the time.

Reply:
in constrast to K9sa I made the `idiot-light` option false by default, to not surprise users after an update.